frontend/src/components/pages/reassign-partitions/components/ActiveReassignments.tsx

```diff
@@ -240,18 +199,71 @@ export class ThrottleDialog extends Component<{ visible: boolean, lastKnownMinTh
                 api.refreshCluster(true);
             });
 
-            msg.setSuccess('Setting throttle rate... done');
+            toast.update(toastRef.current, {
+                status: 'success',
+                description: 'Setting throttle rate... done'
+            })
 
         }
         catch (err) {
             console.error('error in applyBandwidthThrottle: ' + err);
-            msg.setError();
+            toast.update(toastRef.current, {
+                status: 'error',
+                description: 'Setting throttle rate... error'
+            })
         }
 
-        this.props.onClose();
+        onClose();
     }
-}
 
+    return <Modal
+        title="Throttle Settings"
+        open={visible} maskClosable={true} closeIcon={<></>}
+        width="700px"
+
+        onCancel={onClose}
+
+        footer={<div style={{ display: 'flex' }}>
+            <Button
+                variant="outline"
+                colorScheme="red"
+                onClick={() => {
+                    $state.newThrottleValue = null;
+                    void applyBandwidthThrottle();
+                }}
+            >Remove throttle</Button>
+
+            <Button
+                style={{ marginLeft: 'auto' }}
+                onClick={onClose}
+            >Close</Button>
+
+            <Button
+                disabled={noChange}
+                variant="solid"
+                onClick={() => {
+                    void applyBandwidthThrottle();
+                }}
+            >Apply</Button>
+        </div>}
+    >
+        <div style={{ display: 'flex', flexDirection: 'column', gap: '1em', }}>
+            <div style={{ margin: '0 1em' }}>
+                <p style={{ margin: 0 }}>Using throttling you can limit the network traffic for reassignments.</p>
+                <ul style={{ marginTop: '0.5em', padding: '0 1.5em' }}>
+                    <li>Throttling applies to all replication traffic, not just to active reassignments.</li>
+                    <li>Once the reassignment completes you'll have to remove the throttling configuration. <br />
+                        Console will show a warning below the "Current Reassignments" table when there are throttled topics that are no longer being reassigned.
+                    </li>
+                </ul>
+            </div>
```

Comment:
For cases like these, if it's not too much work, using `UnorderedList` and `ListItem` should just work out of the box, so consider it (of course can be a separate PR to avoid scope creep)

Reply:
I think I've already addressed this in the #847, but I'll add a note there to double check